public/app/plugins/datasource/testdata/QueryEditor.tsx

```diff
@@ -138,8 +138,8 @@ export const QueryEditor = ({ query, datasource, onChange, onRunQuery }: Props)
     onUpdate({ ...query, [name]: newValue });
   };
 
-  const onFieldChange = (field: string) => (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
Why the change in the type? This still can be `ChangeEvent<HTMLInputElement>` and use `target` afterwards